exporting/README.md

```diff
@@ -1,8 +1,7 @@
 <!--
----
-title: "Exporting metrics to external databases (experimental)"
+title: "Export metrics to external databases (experimental)"
```

Comment:
@vlvkobal will we keep the `experimental` here?

Reply:
It will be changed in another PR. I'm working on the document right now.

Reply:
Thank you!